src/runners/test_alert_suppressions_runner.py

```diff
@@ -0,0 +1,37 @@
+from runners import alert_suppressions_runner
+from runners.helpers import db
+import os
+import pytest
+
+CTX = db.connect()
```

Comment:
this should be part of a global setup. in general in python, unless you're writing a script that will never be imported, nothing should happen when you import a module